technical-vision.md

```diff
@@ -1,9 +1,28 @@
 # OpenSSF Technical Vision
 
-We envision a future where participants in the open source ecosystem use and share high quality software, with security handled proactively, by default, and as a matter of course:
-- Developers can easily learn secure development practices and are proactively guided by their tools to apply those practices and automatically informed when action is needed to prevent, remediate, or mitigate security issues.
-- Developers, auditors, and regulators can create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.
-- Developers and researchers can identify security issues (including unintentional vulnerabilities and malicious software) and have this information swiftly flow backwards through the supply chain to someone who can rapidly address the issue.
-- Community members can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
+An aspirational view of the future we seek to create in 2-5 years:
 
-The Open Source Security Foundation (OpenSSF) provides tools, services, training, infrastructure, and resources to achieve this vision.
+### OpenSSF is a well-respected strategic partner to large OSS foundations and individual OSS projects
+- OpenSSF is clearly positioned within broader ecosystem with a clear, altruistic charter
+- Periodically review the value proposition of OpenSSF with key stakeholders
+  - Joint OKRs on shared execution, utilization of OpenSSF-managed tools & content
+  - Roadmap alignment: if overlap exists between OpenSSF and another stakeholder, either publicly document rationale OR create/document/execute a convergence plan
```

Comment:
I have no problems with that statement but I think it's worth pointing out that this may raise questions at times, OpenVEX being a case in point.

Reply:
This line initially gave me pause, but I think I've come around to liking it. I'm imagining in practice if the community is not aligned on convergence we'd ask that those projects' readme / documentation include sections on related projects / what's different about their approach.